backend/wmg/data/cube_pipeline.py

```diff
@@ -19,26 +19,26 @@
 logging.basicConfig(level=logging.INFO)
 
 
-def load(dataset_directory: List, group_name: str, validate: bool = False):
+def load(dataset_directory: List, corpus_path: str, validate: bool = False):
     """
     Given the path to a directory containing one or more h5ad files and a group name, call the h5ad loading function
     on all files, loading/concatenating the datasets together under the group name
     """
     with tiledb.scope_ctx(create_ctx()):
         for dataset in os.listdir(dataset_directory):
-            file_path = f"{dataset_directory}/{dataset}/local.h5ad"
+            h5ad_file_path = f"{dataset_directory}/{dataset}/local.h5ad"
             load_h5ad(
-                file_path, group_name, validate
+                h5ad_file_path, corpus_path, validate
             )  # TODO Can this be parallelized? need to be careful handling global indexes but tiledb has a lock I think
             gc.collect()
 
         logger.info("all loaded, now consolidating.")
-        for arr_name in [f"{group_name}/{name}" for name in ["obs", "var", "raw", "X"]]:
```

Comment:
We don't need "X" array anymore, AFAICT.  We only care about the `integrated` array.

Reply:
If so, can also remove the empty array creation from `corpus_schema.py`

Reply:
rm'ing on PR https://github.com/chanzuckerberg/single-cell-data-portal/pull/2377